### PR TITLE
Provide data to contest frames

### PR DESCRIPTION
**Pull request by rbennettcw**
## Link to Issue
Closes: #9865
Closes: #8982

## Description of Changes
- Wires up data for contest frames
  - Home contest frame
  - Eligibility
  - Leaderboard

## Test Plan
- Create Farcaster contest + post frame URL to Warpcast + post content + upvote it
- In terminal, run `frames`, then paste contest frame URL and view in debugger
- (If not setup already) set impersonation account
- Confirm that home + eligibility + leaderboard frames render real contest data

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

**Review by masvelio — COMMENTED** [review]
1. on the app I am able to see the prizes 

image

however on frames prizes are not available

image

File: packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
+import { config } from '@hicommonwealth/core';
+import { models } from '@hicommonwealth/model';
 import { Button } from 'frames.js/express';
 import moment from 'moment';
+import { mustExist } from 'node_modules/@hicommonwealth/model/src/middleware/guards';
+import { getContestBalance } from 'node_modules/@hicommonwealth/model/src/services/commonProtocol/contestHelper';
```
Comment (masvelio) [inline packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx:6]: I am not sure if those imports are correct? node_modules? look above on `config` and `models`

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx [hunk 2]
```diff
@@ -35,12 +37,71 @@ export const contestCard = frames(async (ctx) => {
   // image, title, description, prizes
   // check designs https://www.figma.com/design/NNqlhNPHvn0O96TCBIi6WU/Contests?node-id=960-3689&t=8ogN11dhaRqJP8ET-1
 
-  const prizes = [0.005, 0.003, 0.001];
-
   const contest_address = ctx.url.pathname.split('/')[1];
 
+  const contestManager = await models.ContestManager.findOne({
```
Comment (Rotorsoft) [inline packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx:42]: this should be a query in libs/model I think

**Review by Rotorsoft — APPROVED** [review]

File: packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx [hunk 3]
```diff
@@ -39,20 +40,9 @@ export const contestCard = frames(async (ctx) => {
...
-      {
-        model: models.Community,
-        include: [
-          {
-            model: models.ChainNode.scope('withPrivateData'),
-          },
-        ],
-      },
-    ],
+  const contestManager = await command(GetContest(), {
```
Comment (Rotorsoft) [inline packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx]: Should be query(GetContest()…

Comment (rbennettcw) [reply packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx]: gotcha

**Pull request merged**